OpenSim/Simulation/Model/Appearance.h

```diff
@@ -133,9 +137,9 @@ class OSIMSIMULATION_API Appearance : public Object {
     OpenSim_DECLARE_PROPERTY(color, SimTK::Vec3,
         "The color, (red, green, blue), [0, 1], used to display the geometry. ");
 
-    OpenSim_DECLARE_PROPERTY(surface_properties, SurfaceProperties,
+    OpenSim_DECLARE_UNNAMED_PROPERTY(SurfaceProperties,
         "Visuals applied to surfaces associated with this Appearance.");
-    OpenSim_DECLARE_PROPERTY(curve_properties, CurveProperties,
+    OpenSim_DECLARE_UNNAMED_PROPERTY(CurveProperties,
```

Comment:
Oh, a bonus! This change likely requires updating updateFromXML(); perhaps that can be a separate PR.